swap/src/protocol/alice/message4.rs

```diff
@@ -0,0 +1,101 @@
+use crate::{
+    monero,
+    network::request_response::{Message4Protocol, OneShotCodec, Request, Response, TIMEOUT},
+};
+use libp2p::{
+    request_response::{
+        handler::RequestProtocol, ProtocolSupport, RequestResponse, RequestResponseConfig,
+        RequestResponseEvent, RequestResponseMessage,
+    },
+    swarm::{NetworkBehaviourAction, NetworkBehaviourEventProcess, PollParameters},
+    NetworkBehaviour, PeerId,
+};
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::VecDeque,
+    task::{Context, Poll},
+    time::Duration,
+};
+use tracing::error;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Message4 {
```

Comment:
🙃 What do you think about naming these new messages in a more semantic way? 
I would find that way better for understanding. 
e.g.:

`LockTransferProofMessage` (or similar) for Alice `Message4`
`EncryptionSignatureMessage` (or similar) for Bob `Message5`

(If we do it would apply to the corresponding protocol names - e.g. `Message4Protocol` as well)

Reply:
My 2 sats:

Numbering them scales better because not all messages have a single purpose. In these protocols, you often put whatever you can send at once into a single message which makes it hard to name.

I'd prefer consistency in naming over descriptive names that you can only use sometimes.

However, if you can come up with a descriptive name for all of them, go for it :)